src/debugpy/server/cli.py

```diff
@@ -41,13 +43,14 @@
 
 class Options(object):
     mode = None
-    address = None
+    address: "Optional[tuple[str, int]]" = None
```

Comment:
I think the general feeling in the typing community is that `Foo | None` is more readable than `Optional[Foo]`.

https://typing.readthedocs.io/en/latest/source/modernizing.html#typing-union-and-typing-optional

Reply:
I'll change them to that then.